.env.example

```diff
@@ -0,0 +1,59 @@
+# Do not commit your actual .env file to Git! This may contain secrets or other
+# private information.
+
+# Enable/disable step debug logging (default: `false`). For local debugging, it
+# may be useful to set it to `true`.
+ACTIONS_STEP_DEBUG=true
+
```

Comment:
```suggestion
# Enable/disable step debug logging (default: `false`). For local debugging, it
# may be useful to set it to `true`.
ACTIONS_STEP_DEBUG=false

```

This would produce a lot of extra logging.

Reply:
Disregard, this would be pretty minimal since the actions runtime logs are not included.